src/domain/session/rightpanel/RightPanelViewModel.js

```diff
@@ -1,42 +1,70 @@
 import {ViewModel} from "../../ViewModel.js";
 import {RoomDetailsViewModel} from "./RoomDetailsViewModel.js";
 import {MemberListViewModel} from "./MemberListViewModel.js";
+import {MemberDetailsViewModel} from "./MemberDetailsViewModel.js";
 
 export class RightPanelViewModel extends ViewModel {
     constructor(options) {
         super(options);
         this._room = options.room;
+        this._members = null;
         this._setupNavigation();
     }
 
     get activeViewModel() { return this._activeViewModel; }
 
-    async _getMemberArguments() {
-        const members = await this._room.loadMemberList();
+    async _getMemberListArguments() {
+        if (!this._members) {
+            this._members = await this._room.loadMemberList();
+            this.track(() => this._members.release());
+        }
         const room = this._room;
         const powerLevelsObservable = await this._room.observePowerLevels();
-        return {members, powerLevelsObservable, mediaRepository: room.mediaRepository};
+        return {members: this._members, powerLevelsObservable, mediaRepository: room.mediaRepository};
+    }
+
+    async _getMemberDetailsArguments() {
+        const segment = this.navigation.path.get("member"); 
+        const userId = segment.value;
+        const observableMember = await this._room.observeMember(userId);
+        if (!observableMember) {
+            return false;
+        }
+        const isEncrypted = this._room.isEncrypted;
+        const powerLevelsObservable = await this._room.observePowerLevels();
+        return {observableMember, isEncrypted, powerLevelsObservable, mediaRepository: this._room.mediaRepository};
     }
 
     _setupNavigation() {
         this._hookUpdaterToSegment("details", RoomDetailsViewModel, () => { return {room: this._room}; });
-        this._hookUpdaterToSegment("members", MemberListViewModel, () => this._getMemberArguments());
+        this._hookUpdaterToSegment("members", MemberListViewModel, () => this._getMemberListArguments());
+        this._hookUpdaterToSegment("member", MemberDetailsViewModel, () => this._getMemberDetailsArguments(),
+            () => {
+                // If we fail to create the member details panel, fallback to memberlist
+                const url = `${this.urlCreator.urlUntilSegment("room")}/members`;
+                this.urlCreator.pushUrl(url);
+            }
+        );
     }
 
-    _hookUpdaterToSegment(segment, viewmodel, argCreator) {
+    _hookUpdaterToSegment(segment, viewmodel, argCreator, failCallback) {
         const observable = this.navigation.observe(segment);
-        const updater = this._setupUpdater(segment, viewmodel, argCreator);
+        const updater = this._setupUpdater(segment, viewmodel, argCreator, failCallback);
         this.track(observable.subscribe(() => updater()));
```

Comment:
this can be just fwiw
```suggestion
        this.track(observable.subscribe(updater));
```